common/persistence/sql/sqlplugin/mysql/visibility.go

```diff
@@ -153,7 +160,8 @@ func (mdb *db) SelectFromVisibility(filter *sqlplugin.VisibilityFilter) ([]sqlpl
 			mdb.converter.ToMySQLDateTime(*filter.MinStartTime),
 			mdb.converter.ToMySQLDateTime(*filter.MaxStartTime),
 			*filter.RunID,
-			*filter.MinStartTime,
+			*filter.MaxStartTime,
```

Comment:
@shawnhathaway - this also seemed like a bug to me. can you confirm

Reply:
Yea this does look like a bug as it doesn't appear MinStartTime is even affected by the paging token and this would break the contract of returning earlier records than specified with the filter it appears.